### PR TITLE
[Yamato][7.x.x] Several SDET backports

### PR DESCRIPTION
**Pull request by kivistikliis**
---
### Purpose of this PR
Backports #4710 and #4650

---
### Testing status
[Weekly](https://unity-ci.cds.internal.unity3d.com/project/902/branch/7.x.x%2Fyamato%2Fsdet-backports/jobDefinition/.yamato%2F_abv.yml%23all_project_ci_weekly_2019.4)

---
### Comments to reviewers
- Cache is not backported due to low traffic on this branch
- `Extra` jobs are empty right now, as pipelines for PR/Extra separation were taken as was before (only PR jobs were present). We can decide if we want to keep this layer to keep consistency with the rest of the branches and possibly increase coverage, or remove these redundant layers.
- Compare to the last [weekly ](https://unity-ci.cds.internal.unity3d.com/job/7237576) (it is a few days older and therefore may have some differences)
- This PR enabled HDRP_DXR which was not in the weekly before. Do we want to include it? (It has a few tests failing)

### CODE REVIEW

**Review by sophiaaar — APPROVED** [review]

File: .yamato/_abv.yml [hunk 1]
```diff
@@ -7,51 +7,52 @@
...
+    variables:
+        UTR_VERSION: "current"
+        TEST_FILTER: .*
+    dependencies:
+      -  path: .yamato/_projectcontext.yml#all_package_ci_project_2019.4
          rerun: on_new_revision
-      -  path: .yamato/universal-iphone-metal.yml#Universal_iPhone_Metal_Standalone_il2cpp_Linear_2019.4
+      -  path: .yamato/all-smoke_test.yml#Smoke_Test_2019.4
          rerun: on_new_revision
-    triggers:
```
Comment (theo-pnv) [inline .yamato/_abv.yml:49]: I think the weekly trigger was accidentally removed?

File: .yamato/_projectcontext.yml [hunk 2]
```diff
@@ -420,8 +420,6 @@ all_package_ci_project_2019.4:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  upm-ci package izon -t
       -  upm-ci package izon -d
-    triggers:
```
Comment (theo-pnv) [inline .yamato/_projectcontext.yml:423]: Is this one supposed to go?

Comment (theo-pnv) [reply .yamato/_projectcontext.yml:423]: https://unity.slack.com/archives/G0155CW8683/p1623758613025500

**Review by theo-pnv — CHANGES_REQUESTED** [review]

File: .yamato/_projectcontext.yml (hunk 2 above)
Comment (kivistikliis) [reply .yamato/_projectcontext.yml:423]: Good catch, I don't think so, will add it back now

File: .yamato/_abv.yml (hunk 1 above)
Comment (kivistikliis) [reply .yamato/_abv.yml:49]: Good catch this one too, will push a fix!

**Comment by kivistikliis** [comment]
Last commit
- adds back weekly trigger (using pr_triggers workaround, since the script doesn't yet support per-editor triggers for custom job definitions, see [JIRA](https://jira.unity3d.com/browse/SDET-278) card) 
- changes the general checks trigger to always run on open PRs (to make sure package and smoke tests are run), because on `7.x.x/release` we don't have per-area triggers enabled
- per-area triggers are manually excluded from the commit due to the above-mentioned jira card, and therefore running the script will show changes not included in this PR, which may cause confusion. This will be fixed as soon as possible in a follow-up PR.

File: .yamato/_projectcontext.yml (hunk 2 above)
Comment (kivistikliis) [reply .yamato/_projectcontext.yml:423]: Replaced it with general checks triggers, since this runs both package and smoke tests.

**Review by theo-pnv — APPROVED** [review]

**Comment by kivistikliis** [comment]
Triggers mentioned in previous comment are now fixed in the script (and included in this PR)

**Pull request merged**